src/services/anchor-service.ts

```diff
@@ -188,7 +188,7 @@ export default class AnchorService {
    */
   async _buildMerkleTree(candidates: Candidate[]): Promise<MerkleTree<Candidate>> {
     try {
-      if (config.merkleDepthLimit) {
+      if (config.merkleDepthLimit > 0) {
```

Comment:
Same thing here. We should probably do explicit type conversion. Or should `config` already have this type set as number?

Reply:
`config` *does* have this type set as a number, but it seems to be a string at runtime anyway :(
But `'0' > 0 == false`, so I think this is okay?

Reply:
what if we get `'1'`? is it only a string if 0? Should we parseInt if typeof is string?